### PR TITLE
simplified data to have less nested objects and arrays

### PR DESCRIPTION
**Pull request by jordisman**

### CODE REVIEW

**Comment by ike-okonkwo** [comment]
lgtm

File: database/sampleDataMaker.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const fs = require('fs');
 
 const {
-  type, pageNum, publisher, dates, title, isbn, language, characterArr, awardsArr, editionsArr, settingsArr,
+  type, pageNum, publisher, dates, title, isbn, language, characterArr, awardsArr, editionsArr, settings,
```
Comment (ike-okonkwo) [inline database/sampleDataMaker.js]: Interesting use case - the way you're importing these functions. I would have done it differently.

File: database/sampleDataMaker.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ const createDataObj = () => {
 
 let dataObj = createDataObj();
 
-console.log('dataObj:', dataObj);
+console.log('dataObj = ', dataObj);
```
Comment (ike-okonkwo) [inline database/sampleDataMaker.js]: Are these console.logs necessary or do you just need them for debugging or printing prompts?  You could use logger module for that

Comment (jordisman) [reply database/sampleDataMaker.js]: I used it for testing and see the actual structure of the object

**Pull request merged**